data/json/items/ammo/pebble.json

```diff
@@ -0,0 +1,49 @@
+[
+  {
+    "id": "bearing",
+    "copy-from": "pebble",
+    "type": "AMMO",
+    "name": "bearings",
+    "description": "A box of ball bearings, useful as ammunition for slings.",
+    "weight": 10,
+    "price": 500,
+    "material": "steel",
+    "color": "dark_gray",
+    "count": 50,
+    "range": 15,
+    "damage": 6,
+    "pierce": 1
+  },
+  {
+    "id": "marble",
+    "copy-from": "pebble",
+    "type": "AMMO",
+    "name": "marble",
+    "description": "A handful of glass marbles, useful as ammunition for slings or slingshots.",
+    "weight": 3,
+    "price": 200,
+    "material": "glass",
+    "count": 20,
+    "range": 18,
```

Comment:
> Do range fields in guns have any effect on the range of a firearm anymore?

Yes

> If i drop the range completely from pebble it will have similar range like a unfletched arrow from a selfbow

It should have bad range, probably +1 compared to the `pebbel`. The `wristwrocket` can provide a small further bonus but for distance you're going to need to craft `tihar`